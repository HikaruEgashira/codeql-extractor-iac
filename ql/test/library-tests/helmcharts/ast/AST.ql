private import iac

query predicate helmchart(HelmChart::HelmChartDocument n) { any() }
