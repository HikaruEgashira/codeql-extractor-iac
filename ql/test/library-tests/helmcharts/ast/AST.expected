| Chart.yml:1:1:24:21 | apiVersion: v2 |
